tools/lib/esbuildConfig.js

```diff
@@ -1,58 +1,53 @@
 import fs from "fs/promises";
 import babel from "@babel/core";
 import babelPluginSyntaxTypescript from "@babel/plugin-syntax-typescript";
-import babelPluginSportFunctions from "../babel-plugin-sport-functions/index.cjs";
-import { getSport } from "./buildFuncs.js";
+import babelPluginSportFunctions from "../babel-plugin-sport-functions/index.js";
+import getSport from "./getSport.js";
```

Comment:
I am not sure what is happening here. Something might have changed when I built, but I was unaware of these changes when I made a commit.

Reply:
Generally you should make sure your master branch is updated to the latest version before making a new branch from it. I recommend you do that and then make these changes again on that new branch, since this is not a big PR.